modules/ROOT/pages/cloudhub-api.adoc

```diff
@@ -1,43 +1,85 @@
+** Retrieve statistics for an application or worker.
+** Retrieve application transactions and events for a transaction.
+** Check for replay data, replay the flow for a transaction, and delete replay data for an application.
+** Manage static IP addresses and release unused IP addresses associated with an application.
+** Download the Mule package file for the application.
+** Update, enable, disable, or run all or some schedules.
+
+
+* Manage logs, notifications, and alerts:
+** Add, update, or retrieve log levels for an application.
+** Download the log file for an application or instance.
+** Create, update, retrieve a notification.
+** Mark a notification as read or unread.
+** Create, update, or delete an alert triggered by an application.
+** Retrieve an alert and alert history.
+
+* Manage load balancers:
+** Create, update, delete a load balancer.
+** List load balancers in an organization.
+** Test load balancer rules and return the number of the rule number applied to the input.
+
+* Manage VPCs and VPNs:
+** Create, update, list, and remove a VPC.
+** Create or delete a VPN connection.
+
+* Manage persistent queues
+** Retrieve statistics for persistent queues.
+** Clear the entries from a persistent queue.
+
+* Retrieve additional information about:
+** Supported Mule versions
+** The current organization, including usage statistics
+** Workers, including thread dumps and worker regions
+** Users, including permissions
+** Available domain names
 
-The CloudHub REST API enables you to programmatically access these functions of Runtime Manager:
 
-* Create an application on CloudHub.
-* Deploy a new version of your application.
-* Change the application properties, including the number of workers and environment variables.
-* Get statistics about your application.
-* Create CloudHub notifications.
-* Create email alerts triggered by your applications.
-* Delete your application.
 
 For an interactive reference that includes supported resources, methods, required properties, and expected responses, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/cloudhub-api/[CloudHub API Portal].
 
-The CloudHub API only manages applications deployed to the cloud-based version of Runtime Manager. To manage on-prem applications using an API, see https://anypoint.mulesoft.com/apiplatform/anypoint-platform/#/portals/organizations/ae639f94-da46-42bc-9d51-180ec25cf994/apis/38784/versions/5567732[Runtime Manager API Portal].
+The CloudHub API manages only applications deployed to CloudHub, the cloud-based version of Runtime Manager.
+To manage on-prem applications using the API, see https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/arm-rest-services[Runtime Manager REST Services].
 
 
 == Getting Started and Authenticating with the API
```

Comment:
Going forward, and as updated in the style reference, task topics or task containers cannot use gerunds in their titles, per SFDC and our effort to comply with SFDC...sigh...:

"imperative voice
Use imperative voice in task titles and when telling your audience to do something, such as perform a step:
Build Your Database
Before you can do this, you must create and populate a database with the specifics that you want to track:
. Log in.
. Choose a database template and create an empty file.
. Populate the database."

AND

"title content
Use noun-based headings for conceptual or reference information.
Use an imperative phrase for instruction-based information
In most cases, use plural nouns rather than singular nouns in titles:

	Less preferable	Create an Application
				         Create Application

	Better			Create Applications"

AND

"topic titles
The title of any topic should be a precise and concise statement of what is in the topic. 
Avoid vague titles that include terms like “About,” “Overview,” or “Working with,” or that comprise only the name of a product or feature. 
Where appropriate, identify a user task or goal.
Use a phrase as your title, rather than a question or other complete sentence. 
Do not include both an abbreviation and its spelling. Choose the form most likely to be familiar to your audience, and then present both in the text following the title (see abbreviations).
Google recommends a length of 50-75 characters. "

Reply:
Yup